content/applications/marketing/sms_marketing/essentials/sms_essentials.rst

```diff
@@ -176,6 +175,31 @@ is visualized as different graphs and charts, which can be sorted and grouped in
    :align: center
    :alt: Reporting page in SMS Marketing.
 
+.. tip::
+   SMS messages can be sent using server actions in Odoo. To do so, in :ref:`developer mode
+   <developer-mode>`, navigate to :menuselection:`Settings app --> Technical menu --> Automation -->
+   Automated Actions`. Click :guilabel:`Create` or :guilabel:`New` to create a new action.
```

Comment:
Since this is aimed at 15.0, it should just say "Create," right?
```suggestion
   Automated Actions`. Then, click :guilabel:`Create` to create a new action.
```